packages/web-components/fast-foundation/src/tooltip/tooltip.ts

```diff
@@ -251,6 +259,11 @@ export class Tooltip extends FoundationElement {
      */
     private isAnchorHovered: boolean = false;
 
+    /**
+     * Indicates whether the anchor is currently being focused
+     */
+    private isAnchorFocused: boolean = false;
```

Comment:
You're not using this.  If we don't try to resolve corner cases where someone has mouse over an anchor, tabs to it, then either tabs away or moves the mouse away we can use one flag for both.

Reply:
I've combined the hover and focus properties.